templates/climate_fever/templates.yaml

```diff
@@ -1,5 +1,61 @@
 dataset: climate_fever
 templates:
+  38632cd9-7c4c-4e1d-85b3-20e7a78d4580: !Template
+    id: 38632cd9-7c4c-4e1d-85b3-20e7a78d4580
+    jinja: 'Here''s a statement and accompanying evidence. Does the evidence {{"supports"}},
+      {{"refutes"}}, or provide {{"not enough info"}} on climate change?
+
+
+      Statement: {{claim}}
+
+
+      Evidence: {{evidences[0]["evidence"].strip(".").strip(''"'')}}.
+
+      |||
+
+      {{ ["Supports", "Refutes", "Not enough information"][evidences[0]["evidence_label"]]
+      }}'
+    name: first_evidence_and_claim_itemization
+    reference: First evidence and claim with simple itemization
+  3970f474-a9e3-4264-aefa-dd4cfadd279c: !Template
+    id: 3970f474-a9e3-4264-aefa-dd4cfadd279c
+    jinja: 'Here''s a claim and accompanying evidence statements . Do the statements
```

Comment:
that's the part i am confused about. Each evidence can separately refute, support, etc. the claim.
So how do you create consensus between all evidences (i.e. what is `claim_label`?)
Is it some sort of "do the evidence mostly support the claim?" or "mostly refute the claim?"

Reply:
The claim label per documentation should be assigned with a majority vote over the evidence statements, if I understood correctly . Not sure what is the the exact procedure used to break ties though.

Reply:
I decided to blindly trust the claim_label field 😅but if you think it's better to ignore this, we can simply remove this template.